tests/unit/test_routing_manager.cpp

```diff
@@ -4239,9 +4239,20 @@ void TestBorderRoutingProcessPlatfromGeneratedNd(void)
         AdvanceTime(1000000);
         VerifyPdOmrPrefix(raPrefix);
 
+        sExpectedRios.Add(newRaPrefix);
+
+        // When the prefix is replaced, there will be a short period when the old prefix is still in the netdata, and PD
+        // manager will refuse to request the prefix.
         SendRouterAdvertToBorderRoutingProcessIcmp6Ra(
             {Pio(raPrefix, 0, 0), Pio(newRaPrefix, kValidLitime, kPreferredLifetime)});
-        sExpectedRios.Add(newRaPrefix);
+        for (int i = 5; i < 50; i++)
```

Comment:
This test step is not immedaitely cleer to me.
- We are iterating starting from `i = 5` (why?) and up to `50`.
- And then wait for `1s` and if PD state is "running" we keep sending the same RA to update the PD prefix?

Maybe useful to maybe re-think and update this step? Or maybe add comment explaining it? Thanks.

Reply:
I realized that I can just assume the latency for a stable netdata. Updated.